core-services/ci-secret-generator/_config.yaml

```diff
@@ -87,3 +87,13 @@
   params:
     cluster:
     - app.ci
+- item_name: ci-chat-bot
+  attachments:
+  - cmd: oc --context $(cluster) sa create-kubeconfig --namespace ci $(service_account) | sed "s/$(service_account)/$(cluster)/g"
+    name: sa.$(service_account).$(cluster).config
+  params:
+    cluster:
+    - api.ci
```

Comment:
Why api ci? I thought it only creates prowjobs/talk to app.ci?

Reply:
Copy an paste error.  My bad!  Thanks!